Makefile

```diff
@@ -97,6 +113,51 @@ build-metadata: $(if $(findstring ui,$(BUILD_LABELS)), ui)
 	echo "\n Files: \n"
 	go list -tags $(BUILD_TAGS) -f '{{ join .GoFiles "\n" }}' ./... | sort -u
 
+.PHONY: gen-protobuf
+gen-protobuf: $(PROTOC)
```

Comment:
This can't be run on freebsd.  Add a new target like "check-notfreebsd" similar to what we have now "check-linux" or make this target depend on check-compatibility.